### ISSUE TITLE
Functions on neither onSuccess or onFailure get called.

### ISSUE THREAD

**Issue by debelopumento** [anthonyjgrove/react-google-login#112]
Functions on neither onSuccess or onFailure get called.

<GoogleLogin
          clientId=clinetId
          onSuccess={this.googleLoginSuccess}
          onFailure={this.googleLoginFail}
          isSignedIn={true}
        />

**Comment by anthonyjgrove** [comment]
@debelopumento can you give me an example of your component setup so I can try to reproduce it? Latest is working for me.

**Comment by debelopumento** [comment]
Sure.  There's actually nothing else in the component. But I can try and see if I can make a tiny project for reproducing this issue.  

Thank you,
-Di

**Comment by anthonyjgrove** [comment]
Just paste the component with the props etc. or set it up in the demo file in the repo

**Comment by JonFranchi** [comment]
I have the same issue.  As a quick demo, I just used create-react-app and then modified App.js to look like this.  Worked with your clientID, then with my own it just did nothing at all.  I tried adding another console.log in the responseGoogle with just plain text and it logged nothing, so obviously responseGoogle isn't getting called.

`import React, {Component} from 'react';
import ReactDOM from 'react-dom';
import { GoogleLogin } from 'react-google-login';


const responseGoogle = (response) => {
  console.log(response);
}

class App extends Component {
  render() {
    return (
      <GoogleLogin
    clientId="CLIENT_ID_HERE"
    buttonText="Login"
    onSuccess={responseGoogle}
    onFailure={responseGoogle}
  />
    );
  }
}

export default App;
`
